uasc/secure_channel_instance.go

```diff
@@ -33,11 +33,12 @@ type channelInstance struct {
 	securityTokenID uint32
 	sequenceNumber  uint32
 	algo            *uapolicy.EncryptionAlgorithm
+	maxBodySize     uint32
 
-	messagesSent uint32
-	// messagesReceived uint32
 	bytesSent uint64
 	// bytesReceived    uint64
+	messagesSent uint32
```

Comment:
I assume that you've reordered these to align for atomic.Load/Store. We should mark that with a comment and maybe even move them to the top to make this clear. Not sure how well `go vet` catches these alignment issues these days. In that case we may want to make the `sync.Mutex` explicit, e.g. `mu sync.Mutex`.

Reply:
Yes, it was to fix issues when running on arm32. Running "GOARCH=arm GOARM=6 go vet" did not catch this issue.

Reply:
I usually add a `// atomic.Load/Store` as a comment and if the alignment is important then maybe point that out as well. Problem with putting it at the end is that every new field might break it.